packages/liquidsoap/liquidsoap.2.0.2-1/opam

```diff
@@ -0,0 +1,183 @@
+opam-version: "2.0"
+synopsis: "Swiss-army knife for multimedia streaming"
+description: """\
+Liquidsoap is a powerful and flexible language for describing your
+streams. It offers a rich collection of operators that you can combine
+at will, giving you more power than you need for creating or
+transforming streams. But liquidsoap is still very light and easy to
+use, in the Unix tradition of simple strong components working
+together."""
+maintainer: "romain.beauxis@gmail.com"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/savonet/liquidsoap"
+bug-reports: "https://github.com/savonet/liquidsoap/issues"
+patches: [
+  "patches/fix-last-metadata.diff"
+]
+extra-files: [
+  ["patches/fix-last-metadata.diff" "md5=bcd8f7659e1e935233e025bd39728670"]
+]
```

Comment:
```
Error in liquidsoap.2.0.2-1:              error 53: Mismatching 'extra-files:' field: "patches/fix-last-metadata.diff", "patches"
```
Afaik, the [extra-files](https://opam.ocaml.org/doc/Manual.html#opamfield-extra-files) and [patches](https://opam.ocaml.org/doc/Manual.html#opamfield-patches) must be all in the files folder, subfolders are not allowed.

There is also a permissions issue:
```
2022-01-02 17:04.22: Error in liquidsoap.2.0.2-1: Forbidden permission for file packages/liquidsoap/liquidsoap.2.0.2-1/files/patches. All files should have permissions 644.
```

Reply:
Thanks for looking into this. Changes applied!